src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/GraphDAO.scala

```diff
@@ -89,6 +89,12 @@ trait GraphDAO {
     if (s.contains('.')) throw new RawlsExceptionWithStatusCode(message = s"User-defined string $s should not contain dot characters", statusCode = StatusCodes.BadRequest)
   }
 
+  def validateAttributeName(name: String) = {
+    if (!Attributable.reservedAttributeNames.forall(!_.equalsIgnoreCase(name))) {
```

Comment:
I want to ignore case although I am torn because other attribute names are case sensitive